.github/ISSUE_TEMPLATE/Documentation.yml

```diff
@@ -0,0 +1,25 @@
+name: Documentation
+description: Report an issue related to the SciPy documentation.
+title: "DOC: "
```

Comment:
I would also add something like [your title]. We do get empty PRs/issues

Reply:
On NumPy we do `DOC: <Please write a comprehensive title after the 'DOC: ' prefix>`. We do still get empty titles though :sweat: